lib/components/Body.js

```diff
@@ -28,6 +28,12 @@ function isRegex(x) {
   return !!(x.exec && x.test && x.toString);
 }
 
+function isSet(x) {
```

Comment:
Your question earlier made me think about this a bit more, perhaps this is a better approach:

https://github.com/glenjamin/devcards-js/issues/9#issuecomment-180021654